elements/uranium_isotopes.md

```diff
@@ -0,0 +1,4 @@
+uranium-1
+uranium-tab
+uranium-hmmmmmmm
+uranium2cool4winter_skool
```

Comment:
Hmmm. I don't know these U isotopes. I've suggested the ones I know below. Would you mind clarifying what you meant here? 

```suggestion
uranium-233
uranium-235
uranium-238
uranium-239
```

Reply:
While I can add those in I needed those uranium the previous ones should also remain for other data handling functionality [see here](google.com)